app/scripts/components/common/mapbox/index.tsx

```diff
@@ -267,6 +278,7 @@ function MapboxMapComponent(props: MapboxMapProps, ref) {
           sourceParams={baseLayerResolvedData.sourceParams}
           zoomExtent={baseLayerResolvedData.zoomExtent}
           onStatusChange={onBaseLayerStatusChange}
+          styleLoaded={styleLoaded}
```

Comment:
Adding the `styleLoaded` as a prop leads to the layer itself having to deal with this change, which may be easily forgotten in new layers.
What do you think of only mounting the layer if the `styleLoaded` is true. The effect should be the same.

Additionally, this is also needed for the compare layer.

Reply:
Well that's an excellent suggestion. Thanks